config/v1/types_oauth.go

```diff
@@ -31,6 +31,7 @@ type OAuthSpec struct {
 	// identityProviders is an ordered list of ways for a user to identify themselves.
 	// When this list is empty, no identities are provisioned for users.
 	// +optional
+	// listType=atomic
```

Comment:
It's a tag, so it should be `// +listType=atomic`

Reply:
Ty!

Reply:
I totally miss a discussion why atomic is correct here. I doubt it is.

Reply:
"Making list atomic to prevent GitOps overwrite" – atomic enforces exactly that: overwrite.